### PR TITLE
doc: makefile

### PR DESCRIPTION
**Pull request by AmanMenda**
This one was a little bit more complex.
@blacky-yg, can you please approve the changes before merging this PR ?

### CODE REVIEW

File: docs/tools_cheatsheet/Makefile.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+**A simple Makefile**
+
+```Makefile
+    CC  =	gcc
+
+    NAME  =	binary_name
+
+    CFLAGS  =	 -Wpedantic -pedantic-errors -w  -Wextra  -Wall -Iinclude
```
Comment (blacky-yg) [inline docs/tools_cheatsheet/Makefile.md:8]: You don't need this for a basic makefile
So, you can remove it or simply add one flag and explain it as well.

File: docs/tools_cheatsheet/Makefile.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+    OBJ	=	$(SRC:.c=.o)
+
+    all:  $(NAME)
+
+    $(NAME):	$(OBJ)
+	    $(CC) -c $(OBJ) -o $(NAME)
+
+    clean:
+	    -rm -f $(OBJ)
```
Comment (blacky-yg) [inline docs/tools_cheatsheet/Makefile.md:25]: Remove the dash before `rm`

**Review by blacky-yg — CHANGES_REQUESTED** [review]

File: docs/tools_cheatsheet/Makefile.md (hunk 2 above)
Comment (AmanMenda) [reply docs/tools_cheatsheet/Makefile.md:25]: It means to ignore errors, see [GNU Make](https://www.gnu.org/software/make/manual/html_node/Errors.html). 
Should I remove this anyway?

Comment (blacky-yg) [reply docs/tools_cheatsheet/Makefile.md:25]: I didn't know that 😅. I think that in basic makefile, you should see the errors. So, yes, just remove it.

**Review by blacky-yg — APPROVED** [review]

**Pull request merged**